examples/src/Plugin/GraphQL/Schema/ConfigurableExampleSchema.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Drupal\graphql_examples\Plugin\GraphQL\Schema;
+
+use Drupal\Component\Plugin\ConfigurableInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\PluginFormInterface;
+use Drupal\graphql\GraphQL\ResolverBuilder;
+use Drupal\graphql\GraphQL\ResolverRegistry;
+use Drupal\graphql\Plugin\GraphQL\Schema\SdlSchemaPluginBase;
+
+/**
+ * @Schema(
+ *   id = "config_example",
+ *   name = "Conigurable schema"
+ * )
+ */
+class ConfigurableExampleSchema extends SdlSchemaPluginBase implements ConfigurableInterface, PluginFormInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getResolverRegistry() {
```

Comment:
This plugin should not have any fields on its own. Just return an empty resolver registry here.